bin/hwinj/pycbc_generate_hwinj

```diff
@@ -413,68 +427,6 @@ h_plus, h_cross = get_td_waveform(sim, approximant=name,
 h_plus = pad_timeseries_to_integer_length(h_plus, sample_rate)
 h_cross = pad_timeseries_to_integer_length(h_cross, sample_rate)
 
-# loop over IFOs to calculate sigma
-for ifo in opts.instruments:
-
-    # get Detector instance for IFO
-    det = Detector(ifo)
-
-    # get time delay to detector from center of the Earth
-    time_delay = det.time_delay_from_earth_center(sim.longitude, sim.latitude,
-                                                  sim.geocent_end_time)
-    end_time = sim.geocent_end_time + time_delay
-
-    # get antenna pattern
-    f_plus, f_cross = det.antenna_pattern(sim.longitude, sim.latitude,
-                                          sim.polarization,
-                                          sim.geocent_end_time)
-
-    # calculate strain
-    logging.info('Calculating strain for %s', ifo)
-    strain = f_plus * h_plus + f_cross * h_cross
-
-    # taper waveform
-    logging.info('Tapering strain for %s', ifo)
-    strain = taper_timeseries(strain, tapermethod=sim.taper)
-
-    # FFT strain
-    logging.info('FFT strain for '+ifo+'...')
-    strain_tilde = make_frequency_series(strain)
-
-    # calculate sigma-squared SNR
-    logging.info('Calculating sigma for %s', ifo)
-    sigma_squared = sigmasq(DYN_RANGE_FAC * strain_tilde,
-                            psd=psd_dict[ifo],
-                            low_frequency_cutoff=opts.psd_low_frequency_cutoff,
-                            high_frequency_cutoff=f_high)
-    logging.info('Sigma integrated from %.3f to %.3fHz for %s is %.3f',
-                 opts.psd_low_frequency_cutoff, f_high, ifo,
-                 numpy.sqrt(sigma_squared))
-
-    # populate IFO end time columns
-    setattr(sim, ifo[0].lower()+'_end_time', int(end_time))
-    setattr(sim, ifo[0].lower()+'_end_time_ns', int(end_time % 1 * 1e9))
```

Comment:
We should still set these IFO time variables. Should just have to add lines 419-425, 455-456 to the IFO for-loop above. From past experience, people will see these columns on GraceDB and will not like it if they are not set.